commcare_connect/users/helpers.py

```diff
@@ -14,3 +16,17 @@ def get_organization_for_request(request, view_kwargs):
 
     membership = request.user.memberships.select_related("organization").first()
     return membership.organization if membership else None
+
+
+def create_hq_user(user, domain, app, api_key):
```

Comment:
nit: `app` doesn't seem to be used

Reply:
Ah yep. I was originally going to pull domain from there, then just passed that it in.